CDTDatastoreReplicationAcceptanceTests/ReplicationAcceptance.m

```diff
@@ -1957,6 +1982,69 @@ -(void) testURLConnectionChangeTrackerWithRealRemote
     XCTAssertTrue(changeTrackerGotChanges);
 }
 
+-(void) testURLConnectionChangeTrackerWithRealRemoteAndIAMKey
```

Comment:
What is the particular motivation for this new test, is this code not covered by the RAT?

Reply:
This was added to use `CDTIAMSessionCookieInterceptor` as part of the session in the `TDChangeTracker`.   
The test can be removed if we create `shared_todo_sample` database in the clientlibs-test account and add logic for creating an IAM interceptor in the [original test](https://github.com/cloudant/CDTDatastore/blob/734167816daa208bde38a02796d9dea3b68d24fa/CDTDatastoreReplicationAcceptanceTests/ReplicationAcceptance.m#L1903).

Reply:
@ricellis Are you OK with keeping this test? [Updated link](https://github.com/cloudant/CDTDatastore/pull/388/commits/321959d670eb3a464b87825c542df553f66933a7#diff-172fa1574ebaa7fcd57f7680a6e18c26R1985) since I've rebased.